src/dom/node.ts

```diff
@@ -635,6 +635,9 @@ export class Text extends CharacterData {
   get textContent(): string {
     return <string> this.nodeValue;
   }
+  set textContent(value) {
```

Comment:
Thanks for the PR. Can you put a line space between the two blocks and add a test?

Reply:
Actually, since `Text` extends CharacterData alreday implements getter/setter for textContent: Are you fine with removing both of them from `Text`?